tyk-headless/templates/_helpers.tpl

```diff
@@ -30,3 +30,23 @@ Create chart name and version as used by the chart label.
 {{- define "tyk-headless.chart" -}}
 {{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
+
+{{- define "tyk-headless.redis_url" -}}
+{{- if .Values.redis.addrs -}}
+{{ join "," .Values.redis.addrs }}
+{{- else if and .Values.redis.host .Values.redis.port -}}
+{{ .Values.redis.host }}:{{ .Values.redis.port }}
+{{- else -}}
+redis.{{ .Release.Namespace }}.svc.cluster.local:6379
+{{- end -}}
+{{- end -}}
+
+{{- define "tyk-pro.mongo_url" -}}
```

Comment:
don't need tyk-pro helper in tyk-headless install, right?

Reply:
Good catch! That should say headless